pepdbagent/modules/project.py

```diff
@@ -57,7 +51,8 @@ def get(
         namespace: str,
         name: str,
         tag: str = DEFAULT_TAG,
-        raw: bool = False,
+        raw: bool = True,
```

Comment:
We default to `raw` now? What was the reason for the change?

Reply:
We almost never use processed pep, so I decided to change it to raw